README.md

```diff
@@ -1,2 +1,308 @@
-# bibernate
-Java ORM framework
+# Bibernate
+
+## Java ORM framework
+
+---
+
+## What is Bibernate?
+
+Bibernate is a Java Object-Relational Mapping (ORM) framework which can help you <b>work with a
+database in an object-oriented manner</b>
+
+---
+
+## Quick Start
+
+---
+To install Bibernate locally in your project you should:
+
+* clone repo ```https://github.com/hoverla-bobocode/Bibernate.git```
+* go to the root of Bring project ```cd <path_to_bring>/Bibernate```
+* build jar with ```mvn clean install -DskipTests```
+* add jar to your maven project:
+
+```
+<dependency>
+    <groupId>com.bobocode.hoverla</groupId>
+    <artifactId>bibernate</artifactId>
+    <version>1.0-SNAPSHOT</version> 
+</dependency>
+<dependency>
+    <groupId>com.h2database</groupId>
+    <artifactId>h2</artifactId>
+    <version>2.1.214</version>
+</dependency>
+```
+
+## Supported Databases
+* H2
+* PostgreSQL
+
+## How to use
+
+There are multiple steps you need to perform to correctly use Bibernate:
+
+1. Create `src/main/resources/persistence.yml` [resource file](#persistenceyml-file-example) with information about your
+   database.
+2. Create [entities](#entity-declaration-example), that represent table from the database.
+    * Mark them with the [`@Entity`](src/main/java/com/bobocode/bibernate/annotation/Entity.java) annotation.
+    * Add field that represents primary key of relevant table and annotated with
+      the [`@Id`](src/main/java/com/bobocode/bibernate/annotation/Id.java)
+3. Create [`SessionFactory`](src/main/java/com/bobocode/bibernate/session/SessionFactory.java) that would represent your
+   persistence unit using [`Persistence`](src/main/java/com/bobocode/bibernate/Persistence.java) class
+4. Create [`Session`](src/main/java/com/bobocode/bibernate/session/Session.java object to interact with a persistence context
+   using [`SessionFactory`](src/main/java/com/bobocode/bibernate/session/SessionFactory.java). Look at
+   the [example](#sessionsrcmainjavacombobocodebibernatesessionsessionjava-creation-example) below.
+
+#### `persistence.yml` file example:
+
+```yaml
+logLevel: TRACE
+persistenceUnit:
+  name: default
+  dataSource:
+    jdbcUrl: jdbc:h2:mem:testdb
```

Comment:
maybe it make sence to add some script like was done it tests? just to simplify "demo" ? what do you think?
like : `jdbc:h2:mem:testdb;INIT=RUNSCRIPT FROM 'src/main/resources/products.sql';DB_CLOSE_DELAY=0`